src/Bundler.js

```diff
@@ -448,12 +449,31 @@ class Bundler extends EventEmitter {
     return Server.middleware(this);
   }
 
-  serve(port = 1234) {
-    this.logger.persistent(
-      'Server running at ' + this.logger.chalk.cyan(`http://localhost:${port}`)
-    );
+  async serve(port = 1234) {
```

Comment:
this change will break `--open` option on `bin/parcel`

Reply:
This should now be fixed